__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+process.env.NODE_ENV = "test";
```

Comment:
Don't need this line. NODE_ENV will automatically set to "test" when test is running